website/public/js/controllers/filtersCtrl.js

```diff
@@ -30,6 +31,12 @@ habitrpg.controller("FiltersCtrl", ['$scope', '$rootScope', 'User', 'Shared',
       // User.save();
     };
 
+    $scope.$watch('filterQuery', function (newValue, oldValue) {
```

Comment:
We can assign user to the scope of the controller and then use it's `filterQuery` attribute as a `ng-model` for the search field. I thought that a watcher might be more transparent for the programmers - so they can directly see the relation between user service and filter controller w/o digging into the view and it is easier to test too. I mean, I can still change it if you like!